avaframe/com1DFA/deriveParameterSet.py

```diff
@@ -346,6 +358,24 @@ def splitVariationToArraySteps(value, key, fullCfg):
             itemsArray = np.linspace(0.0, float(itemsL[0]), int(itemsL[1]))
         else:
             itemsArray = np.linspace(-1.*float(itemsL[0]), float(itemsL[0]), int(itemsL[1]))
+    # if variaiton following normal distribution
+    elif 'Dist' in key:
+        itemsArray = []
+        # if not already appended the step of the distribution values that shall be taken as value
+        # add to string so it is clear which value shall be taken from distribution
+        # first check format of string
+        if len(itemsL) == 6:
+            if 'distribution' in itemsL[0]:
+                for i in range(int(itemsL[1])):
+                    itemsArray.append('%d$' % i + value)
+            else:
+                message = ('Format of %s is not correct - required format: \
+                    step$typeOfDistribution$numberOfSteps$ci95value$ci95$support, \
```

Comment:
Not sure what the additional step parameter at the beginning is for? This is nowhere mentioned, at least I cannot find it?!

Reply:
this is needed to decide which step of the distribution is chosen, as the distribution is only build once we are already in the simDict for each single sim - this step is appended inside the function that creates the configuration entry for this, and on the other hand it can be also provided right from the start if only one step is desired

Reply:
splitVariationToArraySteps there it is appended if not provided already but it is right in the error message I removed the step - because it is actually in the if for not providing the step so removed the step there